src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php

```diff
@@ -151,6 +151,11 @@ private function joinRelations(QueryBuilder $queryBuilder, QueryNameGeneratorInt
                 continue;
             }
 
+            if ($mapping['targetEntity'] === $resourceClass) {
+                $queryBuilder->addSelect($associationAlias);
```

Comment:
This works for now, maybe we should just do this instead of the complex `$this->addSelect`. It'll simplify the code, but I'm not sure about the impact on performances.